### ISSUE TITLE
missing Collections.nCopies serializer

### ISSUE THREAD

**Issue by lbergelson** [magro/kryo-serializers#56]
Would it be possible to add a Serializer for the `CopiesList` class that's returned by `Collections.nCopies()`?

One exists already [here](https://www.codatlas.com/github.com/apache/giraph/HEAD/giraph-core/src/main/java/org/apache/giraph/writable/kryo/serializers/CollectionsNCopiesSerializer.java) but it would be convenient if it was included in this package.

**Comment by magro** [comment]
Such a serializer could also go into kryo core since `Collections.nCopies` belongs to the jdk. Just submit a pull request there :-)

**Comment by lbergelson** [comment]
Ah, good point.

**Comment by akiezun** [comment]
how is that different from `CollectionsSingletonListSerializer - for lists created via Collections#singletonList(Object)`? why is that one here and nCopies would be in core kryo?

**Comment by magro** [comment]
The difference is that I initially wrote the `CollectionsSingletonListSerializer` as part of this addon project, later it was added to kryo core (see [DefaultSerializers](https://github.com/EsotericSoftware/kryo/blob/master/src/com/esotericsoftware/kryo/serializers/DefaultSerializers.java#L567)).
At some point we came to the conclusion that kryo can provide any serializer for jdk classes, and serializers for 3rd party libs should go into kryo-serializers. So today `CollectionsSingletonListSerializer` would be added to kryo directly :-)

In kryo-serializers there's still #17 open to remove obsolete serializers. Few other serializers can also be moved to kryo, this is tracked by EsotericSoftware/kryo#151.
